webpack.config.js

```diff
@@ -1,42 +1,41 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
-const fs = require('fs');
```

Comment:
Lint told me to remove this fs.

Reply:
Huh, I'm surprised it's not actually in use, but I can't find where to :+1: